websites/H/HackTheBox/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "937757295453044806",
+		"name": "Atsukoro1#6323"
+	},
+	"service": "HackTheBox",
+	"description": {
+		"en": "Hack The Box is a massive, online cybersecurity training platform, allowing individuals, companies, universities and all kinds of organizations around the world to level up their hacking skills."
+	},
+	"url": "app.hackthebox.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/eNi7hX2.png",
```

Comment:
Logos must be [512x512px at least](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo). Find or create a larger version.